superset-frontend/src/dashboard/components/dnd/DragHandle.tsx

```diff
@@ -17,33 +17,35 @@
  * under the License.
  */
 import React, { LegacyRef } from 'react';
-import cx from 'classnames';
+import { css, styled } from '@superset-ui/core';
+import Icons from 'src/components/Icons';
 
 interface DragHandleProps {
   position: 'left' | 'top';
-  innerRef: LegacyRef<HTMLDivElement> | undefined;
-  dotCount: number;
+  innerRef?: LegacyRef<HTMLDivElement> | undefined;
 }
 
+const DragHandleContainer = styled.div<{ position: 'left' | 'top' }>`
+  ${({ theme, position }) => css`
+    margin-bottom: ${theme.gridUnit}px !important;
```

Comment:
```suggestion
    height: 21px;
```

Reply:
👍 
Looks good after changing to height: 20, thanks for the tip
<img width="613" alt="image" src="https://user-images.githubusercontent.com/15073128/160622851-00e092c7-d313-49f3-93aa-dbf1387a4483.png">